kaggle_environments/envs/lux_ai_2021/todo.md

```diff
@@ -0,0 +1,18 @@
+# TODOs
```

Comment:
Is this list current / which of these need to be done before we launch beta?

Reply:
this is all done with one exception. It would be nice to somehow retrieve the name of a team and store it into the replay file.  If it's not possible then it's fine, people will just need to know if theyre team or team 1.

Reply:
@harrisse is it possible to store team names in the replay?

The rest of these should be added as github issues rather than a todo here. This file can be removed.

Reply:
Yes, for prod episodes you'll get team names in the replay under the top level info property, e.g.:

```"info": {"EpisodeId": 23575003, "LiveVideoPath": null, "TeamNames": ["Goose luck", "Goosebumps", "HandyRL", "NejuMixWATORI"]}```

Reply:
Ok this does not require a change to this PR. This will be a change on our side for the visualizer at https://github.com/Lux-AI-Challenge/LuxViewer2021. At the moment when loading kaggle replays, the visualizer just puts blanks for the names, to be changed soon.